app/assets/javascripts/discourse/app/components/relative-time-picker.js

```diff
@@ -14,19 +14,21 @@ export default Component.extend({
 
   @on("init")
   cloneDuration() {
-    let mins = this.durationMinutes;
-    let hours = this.durationHours;
+    const passedAttributes = Object.keys(this.attrs);
 
-    if (hours && mins) {
+    const usesHours = passedAttributes.includes("durationHours");
+    const usesMinutes = passedAttributes.includes("durationMinutes");
+
+    if (usesHours && usesMinutes) {
       throw new Error(
         "relative-time needs initial duration in hours OR minutes, both are not supported"
       );
     }
 
-    if (hours) {
```

Comment:
We can't rely on presence of the value here, because both `0` and `null` (both legitimate values in some cases) will cause this to go into the wrong conditional branch.